.misc/build_and_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# Get environment variables, readily decrypted by rultor
+#source ../rultor_secrets.sh
+
+if [${UNSUPPORTED} == "false"] || [${TRAVIS_BRANCH} == "master"]; then
```

Comment:
I think this has to be and/`&&` condition given on issue if `[${TRAVIS_BRANCH} != "master"] then also it will run

Reply:
but we want only on master

Reply:
We don't build only if we're not on master and we are not supported, 
let x == 'is not supported'
y == 'is on master'

`x && !y == don't build`

let's see when we do build

`!(x && !y) --by deMorgan theorem--> !x || y`